### PR TITLE
Fixed issue #23841

### PR DESCRIPTION
**Pull request by mahesh-rajawat**
Fixed issue #23841

### Fixed Issues (if relevant)

1.  #23841

### Manual testing scenarios (*)

1. checked with all default eav attribtues.
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Eav.php [hunk 1]
```diff
@@ -787,12 +787,15 @@ public function setupAttributeMeta(ProductAttributeInterface $attribute, $groupC
...
     {
         if ($attribute->getAttributeCode() === 'page_layout') {
-            $defaultValue = $this->scopeConfig->getValue(
-                'web/default_layouts/default_product_layout',
-                \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
-                $this->storeManager->getStore()
-            );
-            $attribute->setDefaultValue($defaultValue);
+            $defaultValue = $attribute->getDefaultValue();
+            if (!$defaultValue) {
```
Comment (Stepa4man) [inline app/code/Magento/Catalog/Ui/DataProvider/Product/Form/Modifier/Eav.php]: Please optimize this code. Nested ifs are looking weird. Why don"t you check it in one if statement?

**Review by Stepa4man — CHANGES_REQUESTED** [review]
Hi @maheshWebkul721
Thanks for your contribution
Please check my comments and tests.

**Review by Stepa4man — CHANGES_REQUESTED** [review]
Hi @maheshWebkul721 
Thanks for your contribution

Please add to all changed files
```php
    declare(strict_types=1);
```
You can read more [here](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#1-basic-programming-principles)

**Comment by engcom-Echo** [comment]
HI @maheshWebkul721 Due to [Magento Definition of Done](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing_dod.html) all code must be covered by tests.

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
Stepa4man, @maheshWebkul721, thank you for your collaboration. 

Please look [comment](https://github.com/magento/magento2/issues/23841#issuecomment-638648987).

**Comment by engcom-Echo** [comment]
Hi @maheshWebkul721, I'm closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for your collaboration

**Pull request closed without merging**